docs/developerguide.rst

```diff
@@ -9,4 +9,43 @@ understand REANA source code and contribute to it.
 Debugging
 ---------
 
-We use ``wdb``.
+Assuming that REANA is being run in development mode following
+`reana-cluster documentation
+<http://reana-cluster.readthedocs.io/en/latest/developerguide.html#deploying-latest-master-branch-versions>`_,
+to debug a component we need to rebuild its Docker image in debug mode. Let say
+we are going to debug `reana-server`:
+
+.. code-block:: console
+
+   $ # if we are not connected to the minikube docker daemon we should do it
+   $ eval "$(minikube docker-env)"
+   $ docker build . --build-arg DEBUG=true -t reanahub/reana-server:v0.1.0
```

Comment:
Hmm, I don't like the possible "overriding" of the `0.1.0` tag here; the tag is supposed to be static.

I'm just using `latest` locally:

```console
$ cd src/reana-cluster
$ cp ./reana_cluster/configurations/reana-cluster-dev.yaml .
$ sed -i -e 's,0.1.0,latest,g' reana-cluster-dev.yaml
$ reana-cluster -f ./reana-cluster-dev.yaml init
```

Shall we change our standard `reana-cluster-dev.yaml` to use `latest` tag by default there?

(and explain in the REANA-Cluster documentation that one can keep versions at some fixed released point, such as `0.1.0`, and change to `latest` only the components you would like to debug)

Reply:
👍 I will make PR to use `latest` tag as default for dev